biojava-structure/src/main/java/org/biojava/nbio/structure/Calc.java

```diff
@@ -763,7 +764,11 @@ public static final void shift(Group group, Atom a) {
 	}
 
 	/**
+<<<<<<< HEAD
+	 * Returns the centroid of the set of atoms.
+=======
```

Comment:
Nah - it's a novel bit shift operation. See Effective Java - Chapter 72,
Section 4. ;-)

On Tue, Aug 30, 2016 at 3:34 PM Jose Manuel Duarte <notifications@github.com>
wrote:

> In biojava-structure/src/main/java/org/biojava/nbio/structure/Calc.java
> <https://github.com/biojava/biojava/pull/571#discussion_r76892590>:
>
> > @@ -763,7 +764,11 @@ public static final void shift(Group group, Atom a) {
> >  	}
> >
> >  	/**
> > +<<<<<<< HEAD
> > +	 * Returns the centroid of the set of atoms.
> > +=======
>
> This looks like something went wrong in merging
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/biojava/biojava/pull/571/files/72c6768ef99e9f6396829d16563971b97c44b8a9#r76892590>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFuRIVXbVXY97YxIPeAlwNvooBOYUT-gks5qlK_wgaJpZM4Jwt87>
> .
>